rerun_cpp/src/rerun/archetypes/transform3d_ext.cpp

```diff
@@ -121,14 +121,15 @@ namespace rerun {
         ///
         /// \param _translation \copydoc Transform3D::translation
         /// \param rotation \copydoc datatypes::TranslationRotationScale3D::rotation
-        /// \param scale \copydoc datatypes::TranslationRotationScale3D::scale
+        /// \param _scale \copydoc Transform3D::scale
         /// \param from_parent \copydoc datatypes::TranslationRotationScale3D::from_parent
         Transform3D(
             const components::Translation3D& _translation, const datatypes::Rotation3D& rotation,
-            const datatypes::Scale3D& scale, bool from_parent = false
+            const components::Scale3D& _scale, bool from_parent = false
```

Comment:
https://github.com/rerun-io/rerun/blob/4c682edec32d9763e66d32d456bd3af5c2b7926d/CODE_STYLE.md#L152-L181


```suggestion
            const components::Scale3D& scale_, bool from_parent = false
```

here and everywhere